vault/raft.go

```diff
@@ -198,11 +165,47 @@ func (c *Core) startRaftBackend(ctx context.Context) (retErr error) {
 }
 
 func (c *Core) setupRaftActiveNode(ctx context.Context) error {
+	raftBackend := c.getRaftBackend()
+	if raftBackend == nil {
+		return nil
+	}
+
+	c.logger.Info("starting raft active node")
+
+	// Read autopilot configuration from storage
+	autopilotConfig, err := c.autopilotConfiguration(ctx)
+	if err != nil {
+		return err
```

Comment:
Might want to log and fall back to default config here instead of failing to come up.

Reply:
Done.